src/service/filtering/filtering_processing.py

```diff
@@ -0,0 +1,168 @@
+from typing import Callable
+
+from fastapi import Request
+
+from src.common.product_models import columnar_attribs_common_models as col_models
+from src.service import errors, app_state
+from src.service.filtering import analyzers
+from src.service.filtering.filters import FilterSet
+from src.service.processing import SubsetSpecification
+
+_FILTER_PREFIX = "filter_"
```

Comment:
```suggestion
FILTER_PREFIX = "filter_"
```
This is in the public API